mmv1/products/filestore/Instance.yaml

```diff
@@ -294,3 +294,48 @@ properties:
     type: String
     description: |
       The reason for enabling deletion protection.
+  - name: 'replication'
```

Comment:
My understanding here is that we set this value on creation and then never set it again from a client. However, can the server modify the (non-output) values under the user?

Depending on the answer we'll mix `immutable` and `ignore_read`, probably. Potentially a CustomizeDiff.